src/Control Tasks/EEPROMControlTask.cpp

```diff
@@ -1,17 +1,40 @@
 #include "EEPROMControlTask.hpp"
 
+EEPROMControlTask::EEPROMControlTask(unsigned int offset)
+    : TimedControlTask<void>(offset)
+{
+}
+
 void EEPROMControlTask::execute()
 {
     if (!sfr::EEPROM::alloted_time_passed) {
-        sfr::EEPROM::eeprom_value = EEPROM.read(0);
+        // The wait time of two hours has not been reached since last read
```

Comment:
This implies that to enter this block, it could not have happened *since* the last read when the better description here is *prior to and at the point of* the last read it could not have occurred